ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -98,6 +98,32 @@ FAN_SS_VSC_REFM4X:
   - zone_air_refrigerant_concentration_sensor_3
   - zone_air_refrigerant_concentration_sensor_4
 
+FAN_SS_STM_RTM:
+  description: "CRAC unit cooling and humidification with compressor and alarm monitoring."
+  is_canonical: true
+  implements:
+  - FAN
```

Comment:
model as a FCU, this is not a FAN.

Reply:
and model it as what its canonical type should be, were all the data available, not just based on what it sends you.

Reply:
Modeled as FCU